### PR TITLE
Add contribution guidelines

### PR DESCRIPTION
**Pull request by oponder**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+
+**We welcome your contributions!**
+
+The best way to contribute changes are Pull Requests. Please note the following before sending PRs:
+
+* Please make one Pull Request per topic/issue. Please avoid putting several non-related issues in a single pull request.
+
+* By sending a Pull Request, you grant us (Giant Swarm) an irrevokable and unlimited license to use the content/code you contribute without any restriction. You accept that your contribution might be published under the license of this repository. Giant Swarm is not obliged to list you as a contributor.
+
+Questions, comments and requests for additions/improvements can be addressed by [adding an issue](issues/new) to this repository.
```
Comment (ZeissS) [inline CONTRIBUTING.md]: that URL should be absolute.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+**We welcome your contributions!**
+
+The best way to contribute changes are Pull Requests. Please note the following before sending PRs:
+
+* Please make one Pull Request per topic/issue. Please avoid putting several non-related issues in a single pull request.
+
+* By sending a Pull Request, you grant us (Giant Swarm) an irrevokable and unlimited license to use the content/code you contribute without any restriction. You accept that your contribution might be published under the license of this repository. Giant Swarm is not obliged to list you as a contributor.
```
Comment (oponder) [inline CONTRIBUTING.md]: Yup, will fix everywhere.

File: CONTRIBUTING.md (hunk 1 above)
Comment (oponder) [inline CONTRIBUTING.md]: Yeah.. will fix everywhere too.

**Comment by ZeissS** [comment]
Thx, otherwise LGTM

**Comment by ZeissS** [comment]
LGTM

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Contributing
+
+**We welcome your contributions!**
+
+The best way to contribute changes are Pull Requests. Please note the following before sending PRs:
+
+* Please make one Pull Request per topic/issue. Please avoid putting several non-related issues in a single pull request.
```
Comment (marians) [inline CONTRIBUTING.md]: putting > addressing?

**Pull request merged**